Description: don't embed Python version in filename of grammar pickle
Author: Jakub Wilk <jwilk@debian.org>
Bug: https://bitbucket.org/birkenfeld/sphinx/issue/641
Forwarded: not-needed
Last-Update: 2011-02-16

--- a/sphinx/pycode/pgen2/driver.py
+++ b/sphinx/pycode/pgen2/driver.py
@@ -120,7 +120,7 @@
         head, tail = os.path.splitext(gt)
         if tail == ".txt":
             tail = ""
-        gp = head + tail + ".".join(map(str, sys.version_info[:2])) + ".pickle"
+        gp = head + tail + ".pickle"
     if force or not _newer(gp, gt):
         logger.info("Generating grammar tables from %s", gt)
         g = pgen.generate_grammar(gt)
